src/Agent/Dialer.vue

```diff
@@ -27,6 +27,9 @@ export default {
       this.lines = await this.$agent.p_mfa('ws_agent', 'lines_out')
     },
     call (line) {
+      if (this.$agent.is_wrapup()) {
+        this.$agent.p_mfa('ws_agent', 'end_wrapup')
```

Comment:
Why do you call end_wrapup? The backend should change agent's state to outgoing (thus 'ending' the wrap-up).

Reply:
when i place a call in wrapup state without ending wrapup nothing happens